pkg/referrerstore/oras/oras.go

```diff
@@ -247,7 +254,7 @@ func (store *orasStore) ListReferrers(ctx context.Context, subjectReference comm
 
 func (store *orasStore) GetBlobContent(ctx context.Context, subjectReference common.Reference, digest digest.Digest) ([]byte, error) {
 	var err error
-	repository, expiry, err := store.createRepository(ctx, subjectReference)
+	repository, expiry, err := store.RepositoryHelpers.createRepository(ctx, subjectReference)
```

Comment:
@binbin-li I had to specifically add this. If I didn't, then in the tests the `orasStore` base `createRepository` implementation was always used even if the `RepositoryHelpers` interface implementation was overriden by the mocking implementation.